### ISSUE TITLE
Job should accept bound methods, too

### ISSUE THREAD

**Issue by kstrauser** [csurfer/pypette#6]
Pypyette Jobs currently insist that their function be an actual FunctionType object. However, class methods are perfectly good callables that work just fine if you lie to a Job about what they are:

```python
from pypette import Job, Pipe

class Foo:
    def method(self):
        print('In method')

f = Foo()
p1 = Pipe('Method pipe is sad')
p1.add_jobs([Job(f.method)])  # Fails: AssertionError: Python function expected

p2 = Pipe('Function pipe is happy')
p2.add_jobs([Job(lambda: f.method())])
p2.run()
```

That's a problem, though, because

- lambda is kind of an ugly hack
- it throws away information about the method (like its name, so that `.graph()` displays just `<lambda>` instead), and
- it doesn't seem actually functionally necessary.

Please broaden Job's typechecking to accept methods, too. Thanks!

**Comment by csurfer** [comment]
@kstrauser: Good point! Thanks for bringing this to my attention. Fixed in v0.0.8. Please update your package and you should have the functionality.

**Comment by kstrauser** [comment]
Awesome. Thanks, @csurfer!
